docs/en/Debugging-Guide.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: global
+title: Debugging Guide
+group: Resources
+---
+
+* Table of Contents
+{:toc}
+
+This page is a collection of high-level guides and tips regarding how to diagnose issues encountered in
+Alluxio.
+
+Note: this doc is not intended to be the full list of Alluxio questions.
+Feel free to post questions on [Alluxio Mailing List](https://groups.google.com/forum/#!forum/alluxio-users).
+
+## Environment
```

Comment:
This should probably go at the bottom since we want the mailing list to be their last resort